CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+
+#### ottr-reports
+
+The [ottr-reports repository](https://github.com/jhudsl/ottr-reports) houses the functionality for reporting the results of the GitHub Action checks on spelling, URLs, and quiz formatting.
+
+#### ottrpal
+
+The [ottrpal repository](https://github.com/jhudsl/ottrpal) provides the functionality to convert an OTTR course to files ready for use in Leanpub and Coursera
+
+#### cow
+
+The [cow repository](https://github.com/jhudsl/cow) isn't officially an OTTR repository, but is a GitHub API wrapper in R which has additional functions to help wrangle GitHub published courses like OTTR courses.
+
+### Documentation
+
+#### ottrproject.org
+
+The [ottrproject.org repository](https://github.com/jhudsl/ottrproject.org) renders the [OTTR documentation website](https://www.ottrproject.org/). The repository is generated from the [OTTR_Template_Website template](https://github.com/jhudsl/OTTR_Template_Website).
+
+#### Wiki
+
+Listed as the "main OTTR guide" on some READMEs, there's an [OTTR wiki](https://github.com/jhudsl/OTTR_Template/wiki).
+
+#### OTTR paper
+
+There's an [OTTR paper](https://pubmed.ncbi.nlm.nih.gov/37207236/) published in the Journal of Statistics and Data Science Education by C. Savonen, C. Wright, A. Hoffman, et al. in 2023. [For information on citing the manuscript, see here](https://www.ottrproject.org/index.html#how-to-cite-ottr)
+
+#### Cheatsheets
+
+TBA
+
+#### Videos
+
+TBA
+
+## How should you suggest changes or alert us about problems?
+
+There are various ways in which you can request or suggest changes to OTTR. Below we discuss ways in which you can make suggestions or requests, but leave the implementation to others or implement the suggestions yourselves.
+
+### You want to make suggestions, but leave the implementation to someone else
+
+If you want to alert us about an issue or make suggestions, you can open a GitHub Issue or submit a response in our OTTR Feedback Google Form.
+
+#### GitHub Issues
+
+Open an issue describing the bug or feature request for the relevant repository using the following links.
+
+| Repository            | Issues Link                                                                 |
+|:---------------------:|:---------------------------------------------------------------------------:|
+| OTTR_Template         | [Open an issue here](https://github.com/jhudsl/OTTR_Template/issues/new/choose)        |
+| OTTR_Quizzes          | [Open an issue here](https://github.com/jhudsl/OTTR_Quizzes/issues/new/choose)         |
+| OTTR_Template_Website | [Open an issue here](https://github.com/jhudsl/OTTR_Template_Website/issues/new/choose)|
+| ottr-reports          | [Open an issue here](https://github.com/jhudsl/ottr-reports/issues/new/choose)         |
+| ottrpal               | [Open an issue here](https://github.com/jhudsl/ottrpal/issues/new/choose)              |
+| cow                   | [Open an issue here](https://github.com/jhudsl/cow/issues/new/choose)                  |
```

Comment:
My goal is that we eventually deprecate `cow` and have its useful functions absorbed by ottrpal, but I haven't had time to do this. If this is something you'd be interested in doing, happy to have you take this one. Up to you.